pages/src/contact.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```

Comment:
The pages folder and contents can be removed.  I merged them into the handlebars already.

Reply:
contact is in the handlebars...needs javascript to hide until contact link is clicked.